src/overlays/actors/ovl_Demo_Tre_Lgt/z_demo_tre_lgt.c

```diff
@@ -147,6 +147,11 @@ s32 DemoTreLgt_PostLimbDraw(GlobalContext* globalCtx, SkelAnimeCurve* skelCurve,
     }
 
     CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_demo_tre_lgt.c", 448);
+
+    //! @bug missing return
+    // If the return value ends up being false (0), the limb won't draw (meaning no limb at all will draw).
+    // In MQ Debug, `Graph_CloseDisps` has the last instruction writing to v0 before this function ends.
+    // That instruction sets v0 to a non-NULL pointer, which is "true", so the limbs get drawn.
```

Comment:
is skelcurve ovveride limb draw different?
normally, return false means that it *wont* override/null the dlist, and true will make it not draw

ill look at the code for it myself to check later, but curious if you know already

Reply:
I didn't notice but yeah apparently the return value is "reversed", nice catch

"normal" skelanime https://github.com/zeldaret/oot/blob/4729ac131a1f3d8ee28f59c41ed8792f51b09040/src/code/z_skelanime.c#L693
skelcurve https://github.com/zeldaret/oot/blob/4729ac131a1f3d8ee28f59c41ed8792f51b09040/src/code/z_fcurve_data_skelanime.c#L104

(so this comment is correct as far as "return true = draw" is concerned)

Reply:
alright cool
we should prob avoid_ub this, apparently this was broken in gcc builds the whole time and i never noticed